src/main/java/com/samoilenko/onlinebookstore/dto/BookRequestDto.java

```diff
@@ -1,12 +1,18 @@
 package com.samoilenko.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
     private String author;
     private String isbn;
```

Comment:
You should also validate the ISBN (International Standard Book Number) as every book must have one and follow the exact pattern. (P.S we have special annotation for it)